spec/settings-panel-spec.coffee

```diff
@@ -31,22 +31,36 @@ describe "SettingsPanel", ->
             type: 'string'
             default: 'zang'
             order: 100
+          enum:
+            title: 'An enum'
+            type: 'string'
+            default: 'one'
+            enum: [
+              ['one', 'One']
```

Comment:
:+1: Makes it even clearer and sets up for making it easier to possibly adding further metadata in the future.